pkg/cmd/client/kubecfg.go

```diff
@@ -590,36 +587,8 @@ func (c *KubeConfig) executeTemplateRequest(method string, client *osclient.Clie
 }
 
 func (c *KubeConfig) executeConfigRequest(method string, clients ClientMappings) bool {
-	if method != "apply" {
-		return false
-	}
-	if len(c.Config) == 0 {
-		glog.Fatal("Need to pass valid configuration file (-c config.json)")
-	}
-
-	clientFunc := func(m *kmeta.RESTMapping) (*resource.Helper, error) {
-		mapping, ok := clients[m.Resource]
-		if !ok {
-			return nil, fmt.Errorf("Unable to provide REST client for %v", m.Resource)
-		}
-		return resource.NewHelper(mapping.Client, m), nil
-	}
-
-	result, err := config.Apply(c.getNamespace(), c.readConfigData(), clientFunc)
-	if err != nil {
-		glog.Fatalf("Error applying the config: %v", err)
-	}
-	for _, itemResult := range result {
-		if len(itemResult.Errors) == 0 {
-			glog.Infof(itemResult.Message)
-			continue
-		}
-		for _, itemError := range itemResult.Errors {
-			glog.Errorf("%v", itemError)
-		}
-	}
-
-	return true
+	glog.Fatalf("DEPRECATED: THe apply command is now deprecated. Use 'cli create' instead.")
```

Comment:
since we're symlinking such that you can run "osc" instead, maybe this message should just say "use create instead".